default.json

```diff
@@ -5,32 +5,11 @@
   "stabilityDays": 3,
   "prCreation": "not-pending",
   "prHourlyLimit": 2,
-  "prConcurrentLimit": 2,
+  "schedule": "on tuesday",
```

Comment:
doesn't this also group all major to thursdays?
Do we want that? (can we even have it differently?)